components/webfield/ReviewerConsole.js

```diff
@@ -392,7 +402,7 @@ const ReviewerConsole = ({ appContext }) => {
               }
             })
             result.groups.forEach((anonGroup) => {
-              if (anonGroup.id.includes(`/Area_Chair_`)) {
+              if (anonGroup.id.includes(`/${areaChairName}_`)) {
```

Comment:
in this case you need to remove the las character if the areaChairName ends with `s`. 

Area_Chairs -> Area_Chair_
Program_Committee -> Program_Committee_

Another option is to pass this as a param: https://github.com/openreview/openreview-py/blob/master/openreview/venue/group.py#L202